build/azure-pipelines.yml

```diff
@@ -862,6 +862,176 @@ stages:
               searchFolder: "tests/Umbraco.Tests.AcceptanceTest/results"
               testRunTitle: "$(Agent.JobName)"
 
+      - job:
+        displayName: E2E Regression Tests (SQL Server)
+        variables:
+          # Connection string
+          CONNECTIONSTRINGS__UMBRACODBDSN: Data Source=(localdb)\MSSQLLocalDB;AttachDbFilename=|DataDirectory|\Umbraco.mdf;Integrated Security=True
+          CONNECTIONSTRINGS__UMBRACODBDSN_PROVIDERNAME: Microsoft.Data.SqlClient
+        condition: eq(dependencies.Build.outputs['A.build.NBGV_PublicRelease'], 'True')
+        strategy:
+          matrix:
+            WindowsPart1Of3:
+              vmImage: "windows-latest"
+              testCommand: "npm run releaseTest -- --shard=1/3"
+            WindowsPart2Of3:
+              vmImage: "windows-latest"
+              testCommand: "npm run releaseTest -- --shard=2/3"
+            WindowsPart3Of3:
+              vmImage: "windows-latest"
+              testCommand: "npm run releaseTest -- --shard=3/3"
```

Comment:
We don't have that many release tagged tests, so 2 runners might be fine, and what about linux?